content/testsuite/testsuite.go

```diff
@@ -934,6 +980,29 @@ func checkStatus(t *testing.T, w content.Writer, expected content.Status, d dige
 	}
 }
 
+func checkNewlyCreated(t *testing.T, w content.Writer, preStart, postStart, preUpdate, postUpdate time.Time) {
+	t.Helper()
+	st, err := w.Status()
+	if err != nil {
+		t.Fatalf("failed to get status: %v", err)
+	}
+
+	wd := w.Digest()
+	if wd != emptyDigest {
+		t.Fatalf("unexpected digest %v, expected %v", wd, emptyDigest)
```

Comment:
"expected %v, got %v" would be more Go-ish

Reply:
yes, but current codes were wrote "unexpected digest %v, expected %v", I guess a full refactory is required if we agree